space/virtual.go

```diff
@@ -0,0 +1,163 @@
+package space
+
+import (
+	"context"
+	"time"
+
+	"github.com/anyproto/any-sync/app"
+	"github.com/anyproto/any-sync/commonspace"
+	"github.com/anyproto/any-sync/commonspace/headsync"
+	"github.com/anyproto/any-sync/commonspace/object/acl/syncacl"
+	"github.com/anyproto/any-sync/commonspace/object/treesyncer"
+	"github.com/anyproto/any-sync/commonspace/objectsync"
+	"github.com/anyproto/any-sync/commonspace/objecttreebuilder"
+	"github.com/anyproto/any-sync/commonspace/spacestorage"
+	"github.com/anyproto/any-sync/commonspace/spacesyncproto"
+	"github.com/anyproto/any-sync/commonspace/syncstatus"
+	"github.com/anyproto/any-sync/net/peer"
+
+	"github.com/anyproto/anytype-heart/pkg/lib/localstore/objectstore"
+)
+
+const name = "virtualSpaceService"
```

Comment:
maybe name it more descriptive?

Reply:
Hmm, may be `virtualSpaceRegister`?

Reply:
I mean const name

Reply:
afais we usually call it `CName`, is it ok? May be we can call it something like `componentName` or `serviceName`?

Reply:
CName is already taken

maybe just virtualSpaceServiceName?

Reply:
Fixed